test/blas_test.hpp

```diff
@@ -178,6 +178,24 @@ class BLAS_Test<blas_test_args<ScalarT_, MetadataT_, ExecutorType_>>
     }
   }
 
+  template <typename DataType,
+            typename value_type = typename DataType::value_type>
+  static void set_const(DataType &vec, size_t _N, value_type c) {
+    for (size_t i = 0; i < _N; ++i) {
+      vec[i] = c;
+    }
```

Comment:
this is std::fill_n

Reply:
Yes - and it turns out that this function isn't even used, so I'm going to remove it.